packages/llvm/llvm.4.0.0/files/install.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -ex
+
+version=${1/%.?/}
+libdir=$2
+
+function llvm_save {
+    cp cmake/modules/AddOCaml.cmake AddOCaml.cmake.save
+}
+
+function llvm_restore {
+    cp AddOCaml.cmake.save cmake/modules/AddOCaml.cmake
+}
+
+function llvm_install {
+    patch -p1 < $1-link.patch
+
+    mkdir build
+    cd build
+
+    cmake -DLLVM_OCAML_OUT_OF_TREE=TRUE -DLLVM_OCAML_INSTALL_PATH="${libdir}" ..
```

Comment:
Ummm, no, that's not what I meant. Can't you just use `-DLLVM_OCAML_INSTALL_PATH="${libdir}/$1"`?